lib/moonshot/stack.rb

```diff
@@ -93,6 +93,18 @@ def status
       end
     end
 
+    def ssh
+      box_id = @config.instance || instances.sample
+      box_ip = instance_ip(box_id)
+      cmd = ['ssh', '-t']
+      cmd << "-i #{@config.identity_file}" if @config.identity_file
+      cmd << "-l #{@config.user}" if @config.user
+      cmd << box_ip
+      cmd << @config.command if @config.command
+      puts "Opening SSH connection to #{box_id} (#{box_ip})..."
+      system(cmd.join(' '))
```

Comment:
In our command we use `exec` instead of `system`. It probably saves some minuscule amount of memory, but seems better since we don't need the use the result of the command for anything.